ironfish-rust/src/keys/view_keys.rs

```diff
@@ -92,6 +92,58 @@ impl IncomingViewKey {
     }
 }
 
+#[derive(Clone)]
+pub struct ViewKey {
+    pub authorizing_key: jubjub::SubgroupPoint,
+    pub nullifier_deriving_key: jubjub::SubgroupPoint,
```

Comment:
It is possible for us to tack on `outgoing_view_key` here. It would also mean we could use a name like `FullViewKey` wonder if the wallet cleanliness is worth the added change? We could also make this change later, which probably is the safest option considering we are close to launch

Reply:
Closing comment, as well will do this as a followup later